fpdcc/ccfp-asset-dashboard
# Allow for a project score of 0

## Overview

Quick fix to allow users to save a project with a score of 0

Closes #246 

## Testing Instructions
* Create/edit a project
* On the project detail page, set all scores to zero and save
* Confirm that the project page reloads, with the success message at the top
